.circleci/config.yml

```diff
@@ -299,8 +303,8 @@ jobs:
           name: deploy to s3
           command: 'aws s3 sync ./dist s3://$BUCKET_NAME/release'
       - run:
-          name: Trigger Windows build
-          command: './scripts/trigger_windows_build.sh ${APPVEYOR_TOKEN} ${CIRCLE_SHA1} release'
+          name: Deploy to Grafana.com
+          command: './scripts/build/publish.sh'
```

Comment:
A bit unclear if these steps are run in parallel or not, but I suppose not and the publish script are reading artifacts from S3, correct? In those cases looks good.

Reply:
@marefr Dependencies between the jobs are defined in the workflow at the bottom of the file. The deployment job depends on the build job so it will have to wait until that (and at this time: every other job) is completed.

https://github.com/grafana/grafana/blob/d59c86ccbbcf1aeaf2f5a8efe7f5d276b1585bae/.circleci/config.yml#L373